decidim-participatory_processes/app/helpers/decidim/participatory_processes/application_helper.rb

```diff
@@ -1,23 +1,12 @@
 # frozen_string_literal: true
 
 module Decidim
-  module ParticipatoryProcesses
-    # Custom helpers, scoped to the participatory processes engine.
+  module Assmblies
```

Comment:
Maybe you wanted to copy this file to assemblies instead of modifying this one? Also, there's a typo.